src/html/runnabear.html

```diff
@@ -1,47 +1,151 @@
 <!-- section -->
 <section id="bears" class="section">
 
-  <h1 class="h1">Runnabear</h1>
-
-  <h2 class="h2">Full Image</h2>
-
+  <h1 class="h1 weight-strong">Runnabear</h1>
+  <h2 class="h2 weight-strong">Full Image</h2>
   <div class="figure-wrapper">
-    <figure class="figure figure-bears">
-      <img src="images/bears/basic-bear.png" height="240" width="180" alt="Basic Bear">
-      <figcaption class="figcaption">Basic Bear</figcaption>
-      <a class="link" href="images/bears/zip/basic-bear.png" download>Download</a>
+    <figure class="grid-block figure figure-bears">
+      <div class="grid-block vertical justify-right">
```

Comment:
Do we need these divs that wrap the figure contents? I think we could remove them and put `vertical` and `justify-right` on the figures themselves.